wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -154,24 +155,40 @@
   });
 
   private final Schema schema;
+  private final ImmutableMap<ProtoType, TypeName> nameToWireJavaName;
   private final ImmutableMap<ProtoType, TypeName> nameToJavaName;
+  private final ImmutableMap<ProtoType, TypeName> protoTypeToAdapter;
   private final boolean emitAndroid;
   private final boolean emitCompact;
 
-  private JavaGenerator(Schema schema, ImmutableMap<ProtoType, TypeName> nameToJavaName,
+  private JavaGenerator(Schema schema, ImmutableMap<ProtoType,
```

Comment:
nit: if you can avoid it, never wrap lines in the middle of a type name like `ImmutableMap<Prototype,`
`TypeName>`. It makes the code difficult to understand.

Even better, never wrap within a single parameter.

Even this is awkward: `ImmutableMap<ProtoType, TypeName>`
`nameToJavaName`.